src/ORM/Hierarchy/Hierarchy.php

```diff
@@ -361,9 +361,7 @@ public static function prepopulate_numchildren_cache($baseClass, $stage, $idList
             '"ParentID"',
             "COUNT(DISTINCT $idColumn) AS \"NumChildren\"",
         ]);
-        $query->setGroupBy([
-            "ParentID
-        "]);
+        $query->setGroupBy(['"ParentID"']);
```

Comment:
It's probably best to use `Convert::symbol2sql("ParentID")` here.

Reply:
Done.